pre-docs/running_bolt_from_docker_image.md

```diff
@@ -0,0 +1,182 @@
+# Puppet-bolt container
+
+Puppet Bolt it available as a docker image. Container images are published to [dockerhub](https://hub.docker.com/r/puppet/puppet-bolt/tags) with tags corresponding to bolt system package and Rubygem versions in addition to a `latest` tag which points at the newest version. This document covers some possible ways to use the container.
+
+## Downloading the image
+
+You can download the latest image from dockerhub with the following command:
+```
+docker pull puppet/puppet-bolt
+```
+
+## Completely stand-alone
+
+When running Bolt from the container, the `localhost` target is the container environment (not the docker host environment). The following example shows running a command against the localhost target in the container.
+```
+cas@cas-ThinkPad-T460p:~/working_dir/docker_bolt$ docker run puppet/puppet-bolt command run 'cat /etc/os-release' -t localhost
```

Comment:
```suggestion
$ docker run puppet/puppet-bolt command run 'cat /etc/os-release' -t localhost
```

Reply:
I think we want `not the Docker host environment` (caps).

Reply:
Recommend shortening the prompt to just `$` to avoid confusion.